specs/core/0_beacon-chain.md

```diff
@@ -482,6 +485,8 @@ The following data structures are defined as [SimpleSerialize (SSZ)](https://git
 
 ```python
 {
+    'randao_reveal': 'bytes96',
```

Comment:
Why `randao_reveal` and `Eth1Data` out of header?
I understand because they are more of "operations", but is a client likely to want the randao reveals for tracking the randomness when building a header chain even if they don't want all the operations?